app/uk/gov/hmrc/gform/auth/AuthService.scala

```diff
@@ -67,6 +72,28 @@ class AuthService(
         performAgent(agentAccess, formTemplate, lang, ggAuthorised(RecoverAuthResult.noop), ifSuccessPerformEnrolment)
     }
 
+  private def performAWSALBAuth()(implicit hc: HeaderCarrier): AuthResult = {
+    val encodedJWT: Seq[(String, String)] = hc.otherHeaders.filter(header => header._1 == "x-amzn-oidc-data")
+
+    encodedJWT.headOption match {
+      case None => AuthBlocked("You are not authorized to access this service")
+      case Some((_, jwt)) => {
+        val decoder = Base64.getDecoder
+        val splitJwt = jwt.split("\\.")
+        if (!(jwt.split("\\.").length == 3)) {
```

Comment:
You probably want to use `splitJwt` instead of repeating `jwt.split("\\.")`

Reply:
Yes ofc